modules/hostap/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
do you want to make this a 2023 copyright?

Reply:
Yes, will do.